runtime/tests/runtime_integration_tests/common.rs

```diff
@@ -309,6 +311,7 @@ pub fn start_rt_test_pqc_model(
         soc_manifest_svn: svn_to_bitmap(args.soc_manifest_svn.unwrap_or(0)),
         soc_manifest_max_svn: args.soc_manifest_max_svn.unwrap_or(127) as u8,
         fw_svn: svn_to_bitmap(production_state.fw_svn),
+        hek_seed: args.hek_seed.unwrap_or([0xABDEu32; 8]),
```

Comment:
Some of these `0xABDE` values were changed to `0xABDE_FC82u32` do you want to change all of them?